crates/sui-graphql-rpc/src/config.rs

```diff
@@ -425,7 +436,7 @@ impl Default for Limits {
             max_query_depth: 20,
             max_query_nodes: 300,
             max_output_nodes: 100_000,
-            max_query_payload_size: 5_000,
+            max_query_payload_size: DEFAULT_MAX_QUERY_PAYLOAD_SIZE,
```

Comment:
Can we re-inline this? I was trying to avoid the duplication of having a constant for the default, and then also the field declaration and the default impl -- it was causing the docs for these fields to get spread through all those places, and was also causing people to import the constant instead of using the default impl (which becomes a problem when they are pulling in multiple of them, because it makes it harder to identify patterns where we maybe want to supply alternate "default" configurations, like the `ci` ones).

Reply:
Sorry about that, forgot to change it now that I change how we think about they mutation payload.